### PR TITLE
Getting GRPC stub as IO

### PR DESCRIPTION
**Pull request by alari**

### CODE REVIEW

File: contract/grpc/src/main/scala/fluence/contract/grpc/client/ContractsCacheClient.scala [hunk 1]
```diff
@@ -53,13 +53,15 @@ class ContractsCacheClient[C: ContractValidate](stub: ContractsCacheStub)(
...
-      binContract ← IO.fromFuture(IO(stub.find(req)))
+      binContract ← IO.fromFuture(stub.map(_.find(req)))
       contract ← codec.decode(binContract)
       // contract from the outside required validation
       _ ← contract.validateME[IO]
     } yield Option(contract)).recover {
       case err ⇒
         logger.warn(s"Finding contract failed, cause=$err", err)
+        logger.warn(err.toString)
+        err.printStackTrace()
```
Comment (DieMyst) [inline contract/grpc/src/main/scala/fluence/contract/grpc/client/ContractsCacheClient.scala]: Print Stacktrace is not good here

File: dataset/node/src/main/scala/fluence/dataset/node/NodePut.scala [hunk 2]
```diff
@@ -73,104 +73,106 @@ class NodePut[F[_]: Async](service: DatasetStorageRpc[F, Observable])(
...
-           */
-          override def nextChildIndex(keys: Array[Key], childsChecksums: Array[Hash]): F[Int] =
-            toF(
-              for {
-                _ ← pushServerAsk(
-                  PutCallback.Callback.NextChildIndex(
-                    AskNextChildIndex(
-                      keys = keys.map(k ⇒ ByteString.copyFrom(k.bytes)),
-                      childsChecksums = childsChecksums.map(c ⇒ ByteString.copyFrom(c.bytes))
+      oldValue ← service
```
Comment (DieMyst) [inline dataset/node/src/main/scala/fluence/dataset/node/NodePut.scala]: is this changes important?

Comment (alari) [reply dataset/node/src/main/scala/fluence/dataset/node/NodePut.scala]: The only important change here is `.to[F]` at the end of the method. However, it forces scalafmt to reformat the whole.

Comment (RicoGit) [reply dataset/node/src/main/scala/fluence/dataset/node/NodePut.scala]: so sad

**Pull request merged**